main/medium/src/reaper.rs

```diff
@@ -1695,6 +1696,116 @@ impl<UsageScope> Reaper<UsageScope> {
         PositionInQuarterNotes::new(qn)
     }
 
+    /// Returns the MIDI tick (ppq) position corresponding to the start of the measure.
```

Comment:
Okay, I'm pedantic ;)

```suggestion
    /// Returns the MIDI tick (PPQ) position corresponding to the start of the measure.
```